CMakeLists.txt

```diff
@@ -298,6 +299,16 @@ if(VTR_ENABLE_VERBOSE)
     message(STATUS "Enabling increased debugging verbosity")
 endif()
 
+#
+# Build for SPEC CPU Benchmark v8
+#
+set(SPEC_CPU_FLAGS "")
+if(SPEC_CPU)
+    #Enable gcov
```

Comment:
Why does this say enable gcov?

Reply:
Comment on what this define is for ... what has to change / what's the purpose?  (spec benchmarking version of vtr; maximizes portability of code and mimimizing the variation in how much work we do on different platforms).